certify-service/src/main/java/io/mosip/certify/services/VCIssuanceServiceImpl.java

```diff
@@ -110,24 +119,25 @@ public CredentialResponse getCredential(CredentialRequest credentialRequest) {
 
     @Override
     public Map<String, Object> getCredentialIssuerMetadata(String version) {
-       if(issuerMetadata.containsKey(version))
-           return issuerMetadata.get(version);
-       return issuerMetadata.get("latest");
+        if(issuerMetadata.containsKey(version))
+            return issuerMetadata.get(version);
+        throw new InvalidRequestException(ErrorConstants.UNSUPPORTED_OPENID_VERSION);
     }
 
     private VCResult<?> getVerifiableCredential(CredentialRequest credentialRequest, CredentialMetadata credentialMetadata,
                                                 String holderId) {
         parsedAccessToken.getClaims().put("accessTokenHash", parsedAccessToken.getAccessTokenHash());
         VCRequestDto vcRequestDto = new VCRequestDto();
         vcRequestDto.setFormat(credentialRequest.getFormat());
-        vcRequestDto.setContext(credentialRequest.getCredential_definition().getContext());
-        vcRequestDto.setType(credentialRequest.getCredential_definition().getType());
-        vcRequestDto.setCredentialSubject(credentialRequest.getCredential_definition().getCredentialSubject());
+
 
         VCResult<?> vcResult = null;
         try {
             switch (credentialRequest.getFormat()) {
```

Comment:
For the duplicate of Code between CertifyIssuanceService and this class, hope there is a jira task for cleanup

Reply:
Can we just add it here, so we can mark this resolved ?

Reply:
https://mosip.atlassian.net/browse/INJICERT-568